src/events/components/DetailView/AttendanceEvent.tsx

```diff
@@ -55,6 +57,8 @@ const AttendanceEvent = ({ event }: IAttendanceEventProps) => {
   const registrationStart = DateTime.fromISO(event.registration_start);
   const registrationEnd = DateTime.fromISO(event.registration_end);
   const cancellationDeadline = DateTime.fromISO(event.unattend_deadline);
+  // @ts-ignore
```

Comment:
Why is this ignored?

Reply:
Because it is not used yet, but it was left as something fore the others to implement

Reply:
Yes, exactly what @oleast said!
I should perhaps have included a comment about that with a TODO: Remove when used.